invite/src/index.d.ts

```diff
@@ -14,27 +14,26 @@
  * limitations under the License.
  */
 
-/**
- * A standard logging interface.
- */
+import Knex from 'knex';
+
+/** A Knex database interface. */
+export type Database = Knex;
```

Comment:
knex provides its own type defs, no? This shouldn't need to be imported

Reply:
I'm trying to alias `Knex` to `Database` so that the other files don't have to explicitly know about `Knex`, but I'm not familiar enough with Typescript to know the right way to do that or if it's even the right thing to do. WDYT?

Reply:
Oh hmm why do you want to alias it? It could make debugging harder.

But if you have a database service, you could have it implement an interface and have other files reference that instead of knex. Looks like that's what `invitation_record.ts` essentially is. (OT but I'd rename that class)

Reply:
I dropped the alias entirely, and just use Knex as the type now. Using this method with the type declarations, I get no issues compiling the Typescript with `tsc`, but for some reason it fails when I try to actually run the tests. I'm not really sure what's going wrong :disappointed: